mctp-estack/src/lib.rs

```diff
@@ -30,6 +30,11 @@
 #![allow(clippy::int_plus_one)]
 #![allow(clippy::too_many_arguments)]
 
+// clippy may interpret defmt's log macros as those of the log crate, but
```

Comment:
The comment isn't quite right - it does recognise when it's `log::warn` vs `defmt::warn`. But we can't act on the advice when it tells us to change `log::` ones.